src/vmm/tests/integration_tests.rs

```diff
@@ -50,10 +49,50 @@ fn wait_vmm_child_process(vmm_pid: i32) {
 }
 
 #[test]
-fn test_dummy_vmm() {
+fn test_dummy_vmm_elf() {
```

Comment:
This test is very similar to `test_dummy_vmm_bzimage`. Can you create a helper function something like `run_vmm` that does most of the things besides creating the `vmm_config`, and then have the two tests calling into the helper function?